app/models/miq_expression.rb

```diff
@@ -430,11 +430,11 @@ def self._to_human(exp, options = {})
 
   def to_ruby(tz = nil)
     tz ||= "UTC"
-    @ruby ||= self.class._to_ruby(@exp.deep_clone, @context_type, tz)
+    @ruby ||= _to_ruby(@exp.deep_clone, @context_type, tz)
     @ruby.dup
   end
 
-  def self._to_ruby(exp, context_type, tz)
+  def _to_ruby(exp, context_type, tz)
```

Comment:
The same should be done with context_type parameter. Also, other instance methods have this shadowing of attr_reader for @exp and @content_type and need to be changed